Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -21,6 +21,12 @@ compatible with workspaces is described in the
 TCA reference (in the :ref:`description of the "ctrl" section <t3tca:ctrl>`
 and in the :ref:`description of the "versioningWS" property <t3tca:ctrl-reference-versioningws>`).
 
+You might want to turn the workspace off for certain tables. 
+You can do so in the :file:`AdditionalConfiguration.php`::
+
+   $GLOBALS['TCA']['example_table']['ctrl']['versioningWS'] = false;
```

Comment:
We should also note that this will lead to all `t3ver_*` fields of the table to be marked as obsolete if they have not be defined explicitly in the extension. A careless subsequent DB schema update will then drop these fields.

See https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.3/Feature-85160-AutoCreateManagementDBFieldsFromTCACtrl.html